src/pages/search/project_search/ProjectListItem.tsx

```diff
@@ -64,6 +64,7 @@ export const ProjectListItem = ({ project, showEdit = false, onDelete, deleteToo
       <>
         {showEdit && (
           <EditIconButton
+            data-testid={dataTestId.myPage.editProjectButton(project.id)}
```

Comment:
Merk at `project.id` er hele URIen, ikke bare identifier som jeg ser du har brukt i `PersonTableRow`. Blir litt ryddigere med bare identifier i stedet for hele URI, men om det ikke gjør noe for testene går det for det samme